client/src/pages/VocabDrawer/VocabDrawer.js

```diff
@@ -377,7 +425,7 @@ const VocabDrawer = ({ navigation }) => {
       },
       {
         text: 'Remove Image',
-        onPress: () => setImage(null),
+        onPress: () => clearImage(),
```

Comment:
When a user removes an image, we shouldn't delete the image from AWS right away. We should only delete the image after they user saves the changes on the VocabDrawer.